packages/loaders/package.json

```diff
@@ -23,11 +23,13 @@
     "lib",
     "dist"
   ],
+  "peerDependencies": {
+    "@pixi/spritesheet": "5.2.1"
```

Comment:
We need a way to decouple spritesheet and loaders. Using a peerDependency is not the solution. Creating a shared typing or interface is better.

Reply:
i tell you guys all reary that i do not want to see nothing about you guys in mu emails address stop do your onw cloud please migmu11@icloud.com is  the orilligal address and migmu2008@outlook.com is  my  [miguel-murcia] owner